renderer/components/traffic-lights.js

```diff
@@ -0,0 +1,116 @@
+
+import {remote} from 'electron';
+import React from 'react';
+
+export default () => (
+  <div className="root">
+    <div className="traffic-lights">
+      <button type="button" className="traffic-light traffic-light-close" onClick={() => remote.app.quit()} id="close"/>
+      <button type="button" className="traffic-light traffic-light-minimize" onClick={() => remote.BrowserWindow.getFocusedWindow().minimize()} id="minimize"/>
+      <button type="button" disabled className="traffic-light traffic-light-maximize" onClick={() => false && remote.BrowserWindow.getFocusedWindow().maximize()} id="maximize"/>
```

Comment:
Check out the `customButtonsOnHover` BrowserWindow option: https://electronjs.org/docs/api/browser-window#new-browserwindowoptions

Reply:
Nice! But if we use that we can't animate the buttons down, I guess that's OK?
![kapture 2018-05-15 at 22 05 37](https://user-images.githubusercontent.com/5027156/40080714-1b751388-588c-11e8-803d-f3c1b347c1ad.gif)

Reply:
I tried using it, but doesn't seem to work 🤔 

putting `customButtonsOnHover Boolean (optional)` as the `titleBarStyle String (optional)` also doesn't make sense, I tried sending an object and setting `customButtonsOnHover` directly, but nothing worked, I will check issues for this too if we want to use it.

Reply:
The custom implementation was always quirky, so I still think it's worth it if that's all we'll be missing.

Reply:
I think we should ship this implementation in the beta and see what people think, I think it really feels like the native thing but also looks really good.